packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -28,11 +28,11 @@ const Series = memo(
     const seriesProps = { id, data, type, visible, ...restProps };
 
     /*
-  if (defaultTo(restProps.requiresAxis, true)) {
-    const axis = getAxis();
-    if(!axis) throw new Error(`Series type="${restProps.type}" should be wrapped inside Axis`);
-  }
-*/
+      if (defaultTo(restProps.requiresAxis, true)) {
+        const axis = getAxis();
+        if(!axis) throw new Error(`Series type="${restProps.type}" should be wrapped inside Axis`);
+      }
+    */
```

Comment:
Looks like this should be added back?

Reply:
It is disabled because useAxis can actually return null. That is when the axis has yet to be created.

I could not figure how to check that no axis will ever be created. Maybe rewriting whole of axis creation to use events, but that would be quite a task.

Some kind of error would be really nice to help debugging.